async-nats/src/client.rs

```diff
@@ -69,7 +69,7 @@ pub struct Client {
     pub(crate) sender: mpsc::Sender<Command>,
     next_subscription_id: Arc<AtomicU64>,
     subscription_capacity: usize,
-    inbox_prefix: String,
+    inbox_prefix: Arc<String>,
```

Comment:
```suggestion
    inbox_prefix: Arc<str>,
```

`Arc<str>` is generally a more appropriate type for this use case